docs/test/walkthrough-creating-and-running-unit-tests-for-managed-code.md

```diff
@@ -207,10 +207,11 @@ namespace BankTests
 
 ### Add a using statement
 
-Add a [`using` statement](/dotnet/csharp/language-reference/keywords/using-statement) to the test class to be able to call into the project under test without using fully qualified names. At the top of the class file, add:
+Add a [`using` statement](/dotnet/csharp/language-reference/keywords/using-statement) to the test class to be able to call into the project under test (here, the `BankAccount` class) without using fully qualified names. We also add a `System` namespace for calling the `ArgumentOutOfRangeException` when we [create and run new test methods](#create-and-run-new-test-methods) below. At the top of the class file, add:
```

Comment:
@mavaddat Thanks, I tried this and you're correct, we do need this using statement. However, I don't really like adding it in this section that comes way before where it's used (it makes it harder to manage the page going forward, too). Could I ask you to back out this change and instead change line 334 to use `System.ArgumentOutOfRangeException` instead of just `ArgumentOutOfRangeException`?

Reply:
Sounds good. I will resolve the conversation and suggest a new pull on that line 334 you suggested.